persist/fs/files.go

```diff
@@ -42,11 +42,105 @@ var timeZero time.Time
 
 const (
 	dataDirName       = "data"
+	snapshotDirName   = "snapshots"
 	commitLogsDirName = "commitlogs"
 )
 
 type fileOpener func(filePath string) (*os.File, error)
 
+// FilesetFile represents a set of Fileset files for a given block start
+type FilesetFile struct {
+	BlockStart time.Time
+	Files      []string
+}
+
+// FilesetFilesSlice is a slice of FilesetFile
+type FilesetFilesSlice []FilesetFile
+
+// Filepaths flattens a slice of FilesetFiles to a single slice of filepaths
+func (f FilesetFilesSlice) Filepaths() []string {
```

Comment:
Is this absolute or relative paths?  Might be worth mentioning in the method comment or name it `AbsoluteFilepaths(..)` or something.

Reply:
sure